packages/modules/desktop-viewer-react/src/hooks/useDesktopViewerInitializer.tsx

```diff
@@ -16,7 +16,7 @@ import type { DesktopViewerProps } from "../types";
 
 export const useDesktopViewerInitializer = (options?: DesktopViewerProps) => {
   const [desktopViewerInitOptions, setDesktopViewerInitOptions] =
-    useState<ItwinViewerInitializerParams>();
+    useState<Partial<ItwinViewerInitializerParams>>();
```

Comment:
why partial here?

Reply:
good point, makes no difference here since all the props are optional, removed